.github/workflows/build_and_test.yml

```
@@ -10,6 +10,8 @@ on:
         required: false
         default: false
         type: boolean
+  push:
+    branches: ["main", "release-*"]
```

Future changes to the list might be easier with:

```suggestion
    branches: 
     - main
     - "release-*"
```